### ISSUE TITLE
Compilation errors in Swift 4.2

### ISSUE THREAD

**Issue by frangulyan** [wistia/WistiaKit#97]
I got compilation errors in WistiaStatsManager class saying _'willResignActiveNotification' has been renamed to 'NSNotification.Name.UIApplicationWillResignActive'_ (which is strange, since the opposite happened in swift 4.2). I have also noticed that your github repo's build also failed - https://travis-ci.org/wistia/WistiaKit/builds/431507522?utm_source=github_status&utm_medium=notification

Could you please fix this? We are planning to release our app in the next days...

**Comment by frangulyan** [comment]
My podfile has:

    pod 'WistiaKitCore', :git => 'https://github.com/wistia/WistiaKit.git', :branch => 'swift4_2'
    pod 'WistiaKit', :git => 'https://github.com/wistia/WistiaKit.git', :branch => 'swift4_2'

**Comment by Yonodactyl** [comment]
iOS 4.2 requires you to write application states as follows:

> UIApplication.willResignActiveNotification

I don't have the ability to do a pull request but thats what it would need (seemingly).

**Comment by frangulyan** [comment]
Well that's what I wrote - the code in the branch seems to be correct swift 4.2 code, but for some reason it doesn't build, and xcode reports reverse changes, saying "XXX (4.2 version) has been renamed to YYY (4.1 version)". I have no idea if my pods or project setup is wrong, or something is messed in the branch itself. I just noticed, that Travis build here fails with the same message. At least I AlamofireImage dependency should be updated, I have opened another issue on that.
